### DISCUSSION TITLE
Russian? translation request.

### DISCUSSION THREAD

**Discussion by what-a** [iptv-org/community#145] [Q&A] [answered]
Could some kind soul translate this into English for my language impaired brain?  I'm guessing it says something about a station's broadcast stream being temporarily offline (but would like to know for sure).

![image](https://user-images.githubusercontent.com/88639906/159818104-66574be8-7a7b-449a-8dd3-9ba1c25f618e.png)

**Comment by freearhey** [comment, marked as answer]
Your guess is correct

<img width="1265" alt="image" src="https://user-images.githubusercontent.com/7253922/159885807-c66ae8f4-8e82-4d06-b8a1-9ed7761d45c1.png">

https://translate.yandex.com/ocr

**Reply by what-a** [reply]
@freearhey Thank you!
